controllers/api.js

```diff
@@ -287,15 +287,18 @@ exports.postTwitter = (req, res, next) => {
 exports.getSteam = (req, res, next) => {
   const steamId = req.user.steam;
   const params = { l: 'english', steamid: steamId, key: process.env.STEAM_KEY };
-  const playerAchievements = () => {
-    //get the list of the recently played games, and pick the most recent one and get its achievements
-    return request.getAsync({ url: 'http://api.steampowered.com/IPlayerService/GetRecentlyPlayedGames/v0001/', qs: params, json: true })
+  const playerAchievements = () =>
+    /**
+     * get the list of the recently played games,
+     * and pick the most recent one and get its achievements
+     */
```

Comment:
Was this comment getting broken up caused by eslint?

Reply:
It looks like the lit rules marked this because the comment line was more than 100 char long.  It was 103 char long.  I would rather just change one of the `and`sto `&` and make it conform to the 100 char limit than split it into multiple lines in order to keep the code compact.